content/english/useless-box/build-the-box.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Build the box"
+description: ""
+draft: true
+weight: 3
+---
+
+## Let�s build the box
+![diagram of the parts to build the box, base, lids, sides, end, motor mount and finger](img/boxPartsDiagram.png)
+1.	Glue 2 fingers together, to make a thicker finger. (If it�s already thick enough, skip this step)
```

Comment:
Is it possible to have pictures for assembly of each step?

Reply:
yes, we'll be taking pictures and updating this page, we wanted to put some initial content in a first PR, so it gives some shape and can be collaborated on.